samples/dotnet/kernel-syntax-examples/Example21_ChatGPTPlugins.cs

```diff
@@ -17,14 +18,16 @@ public static async Task RunAsync()
     private static async Task RunChatGptPluginAsync()
     {
         var kernel = new KernelBuilder().WithLogger(ConsoleLogger.Log).Build();
+        using HttpClient importHttpClient = new HttpClient();
+        importHttpClient.DefaultRequestHeaders.Add("User-Agent", "Microsoft.SemanticKernel");
```

Comment:
Should be consistent in the value we use for user agent. Looks like elsewhere in this PR we are using "Microsoft-Semantic-Kernel"

Reply:
Good catch!